### DISCUSSION TITLE
keep Budgie of Budgie Edition Up To Date

### DISCUSSION THREAD

**Discussion by AlphaElwedritsch** [SpiralLinux/SpiralLinux-project#379] [Q&A] [answered]
Hi
Is it possible to keep the Budgie version up to date? 10.8 instead of 10.7
With your own repo or some other option, or is that not a good idea? And should or must you refer to the Debian repos and live with the 10.7 Budgie version?

cheers
Alpha

**Comment by geckolinux** [comment, marked as answer]
Hi there, no, creating packages and package repositories is outside of the scope of the SpiralLinux project.

> # https://spirallinux.github.io
> SpiralLinux serves as an alternative live installation method for a highly reliable customized Debian system using only official Debian package repositories.
> 
> - Depends entirely on the Debian infrastructure, thus avoiding the "developer-hit-by-a-bus" concern
